dask_sql/mappings.py

```diff
@@ -137,6 +139,10 @@ def sql_to_python_value(sql_type: "SqlTypeName", literal_value: Any) -> Any:
             literal_value = literal_value.rstrip("'")
             literal_value = literal_value.encode(encoding=encoding)
             return literal_value.decode(encoding=encoding)
+        # If we have a datetime without a date,
+        # then we should offset relative to the Unix epoch
+        elif ":" in literal_value and len(literal_value.split(" ")) == 1:
+            return "1970-01-01 " + literal_value
```

Comment:
I found that hardcoding this was easier than trying to subtract today's date with a timedelta.

Reply:
Should also maybe add some sort of `is_datetime` check.

Reply:
is there a specific case/test which was failing without this?

Reply:
I found that without the `1970-01-10`, it automatically displays `SELECT TIME '08:08:00.091' AS "T"` with the current date, such as `2022-10-03 08:08:00.091`, even though the test is expecting a datetime relative to the Unix epoch.

I do think I should probably add some sort of check that the value is convertible to a datetime, just on the chance that `literal_value` is some random string that has a colon and space in it.

Reply:
Actually, I think the combination of these changes and Andy's changes in #825 removes the need for this `elif` block. I'll remove it and we can check again after his PR is merged.